### ISSUE TITLE
[cdl] Bindings for Python

### ISSUE THREAD

**Issue by radumarias** [xoriors/rencfs#203]
We need some bindings so we can access the lib from Python.
We need to expose public methods (and all related) from [EncryptedFs](https://github.com/xoriors/rencfs/blob/37bada57e2ab6d9d365dd66e76b5d838b8372e1a/src/encryptedfs.rs#L548) so devs could use this similarly like we use it in FUSE implementation.

You can use [pyo3](https://pyo3.rs).

You can use these as ref
- https://github.com/radumarias/rencrypt-python
- https://github.com/radumarias/zeroize-python

Or you can use [SWIG](https://www.swig.org/) like [this](https://chatgpt.com/share/6754645d-1738-8003-8010-063cfb37d939) or how it's used in #202.

**Comment by Nandini1071** [comment]
Hi @radumarias is this issue still open, if yes then what is the requirement would like to contribute on this.

**Comment by radumarias** [comment]
> Hi @radumarias is this issue still open, if yes then what is the requirement would like to contribute on this.

@Nandini1071 yes it is, you can do something similar to https://github.com/radumarias/rencrypt-python and exposing methods like https://github.com/radumarias/rencfs/blob/main/java-bridge/src/lib.rs

**Comment by Nandini1071** [comment]
Ok so what is required I mean could you tell me the exact changes which are required and the file path in GitHub.

**Comment by radumarias** [comment]
@Nandini1071 see the links in the previous comment

**Comment by radumarias** [comment]
@Nandini1071, are you still interested in this?

**Comment by Nandini1071** [comment]
Hi @radumarias , as of now no. Thanks.

**Comment by radumarias** [comment]
@gabrielrusanescu will work on this
